conf/newrelic.ini

```diff
@@ -0,0 +1,210 @@
+# ---------------------------------------------------------------------------
```

Comment:
All provided by the New Relic website, and then tweaked by me to:

 * Blank the name and license
 * Turn some things off (not to the extent we did with the websocket, but that might be a good idea)